simp/simp-core
# (SIMP-9978) simp-core Update acceptance tests for EL7 SIMP server

Test updates:
- General updates
  - Use simp_grub::password in lieu of OBE grub::password in answers
    file passed into `simp config`
  - Disable puppet agent timer appropriately
  - Use postfix instead of haveged as the service to restart in the
    rsyslog forwarding tests, because SIMP masks haveged on EL8
  - Use generic/centos8 and generic/oracle7 in nodesets
  - Try to address random SUT ssh connection closures
    - Adjust nodeset ssh connection parameters
    - Add more places in which we attempt to detect SUT ssh connection
      closures and reconnect
- ipa test suite updates
  - Ensure at least 1 IPv6 address is configured on the IPA server,
    as it fails to install and run otherwise
  - Disable IPA configuration of NTP during ipa-server-install
    and ipa-client-install, as NTP is managed elsewhere in Puppet
- simp_lite test suite update
  - Ensure the iptables and firewalld services are not running on all
    hosts prior to configuration via Puppet. This is necessary to
    create a deterministic test, because on some virtual machines,
    a firewall is enabled by default (e.g., firewalld is enabled on
    generic/centos8 vbox).

Other maintentance:
- Add the default location of the 'dev' GPG key to .gitignore.

SIMP-9978 #close
